ModernWallet/Classes/Extension/UserDefaults+Extensions.swift

```diff
@@ -14,9 +14,15 @@ extension UserDefaults {
         tempEmail: "tempEmail",
         tempPhone: "tempPhone",
         signUpStep: "SignUpStep",
-        loggedIn: "loggedIn"
+        loggedIn: "loggedIn",
+        menuOrder: "menuOrder"
     )
     
+    var menuOrder: [Int] {
+        get { return array(forKey: UserDefaults.keys.menuOrder) as? [Int] ?? [0, 1, 2, 3, 4, 5, 6, 7, 8, 9] }
```

Comment:
I don't think you should unwrap the array here because you don't know what's the number of items in `MenuTableViewController`. You can just go with an optional and unwrap it where you use it. If `nil`, use the count of `items` in `MenuTableViewController`: `menuOrder ?? Array(0...items.count)`